validation_pipeline.ipynb

```diff
@@ -426,15 +425,25 @@
         "# elements = ['d18O_cel', 'd15N_wood', 'd13C_wood']\n",
         "# isotope_column_names = ['d18O_cel', 'd15N_wood', 'd13C_wood']\n",
         "elements = ['d18O_cel']\n",
-        "isotope_column_names = ['d18O_cel']\n",
-        "mean_isoscapes = [\n",
+        "isotope_column_names = ['d18O_cel']"
+      ],
+      "metadata": {
+        "id": "-CHsCJuw8BF4"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "mean_isoscapes_1 = [\n",
```

Comment:
nit: replace mean_isoscapes_1 with mean_isoscapes_a
and mean_isoscapes_2 with mean_isoscapes_b etc

subtle but using a vs b instead of 1 vs 2 makes it much clearer that these two variables are the subject of comparison